var/spack/repos/builtin/packages/ntpoly/package.py

```diff
@@ -26,4 +26,7 @@ class Ntpoly(CMakePackage):
 
     def cmake_args(self):
         args = ["-DNOSWIG=Yes"]
+        if self.spec.satisfies('%fj'):
+            args.append('-DCMAKE_Fortran_MODDIR_FLAG="-M"')
```

Comment:
Quotes shouldn't be necessary here, and occasionally confuse things. There is no shell parser that removes the quotes and separates spaces.
```suggestion
            args.append('-DCMAKE_Fortran_MODDIR_FLAG=-M')
```

Reply:
Sorry for misuse. I fixed it.